experimental/sql/tools/rel_opt.py

```diff
@@ -3,8 +3,15 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+try:
+  from tools.IteratorsMlirConverter import IteratorsMLIRConverter
```

Comment:
What should the capitalization of the file name be? This line shows that the capitalization is not consistent...